README.md

```diff
@@ -119,6 +119,44 @@ interact with the webapp.
 bin/dc-logs -f
 ```
 
+### Database
+
+The database, named `new_to_nl_development` would be accessible on the
+port `5345` in the local host.  By default, the username is `postgres`
+and the password is namesake.  You can explore the database using
+`pssql`:
+
+```bash
+psql -h localhost new_to_nl_development postgres
+```
+
 ### Check the Email Inbox
 
 Visit `http://localhost:8026`
+
+### Pulling new code
+
+After pulling new code with `git pull` or similar, it is possible that
```

Comment:
I love this section! It's a great addition to the docs. Thank you.

Reply:
Indeed.